shell/config/router/navigation-guards/clusters.js

```diff
@@ -73,7 +33,7 @@ export default async function({
     const oldProduct = getProductFromRoute(from);
 
     // Leave an old pkg where we weren't before?
-    const oldPkgPlugin = oldPkg ? Object.values($plugin.getPlugins()).find((p) => p.name === oldPkg) : null;
+    const oldPkgPlugin = oldPkg ? Object.values(store.$plugin.getPlugins()).find((p) => p.name === oldPkg) : null;
```

Comment:
Long term I don't think we want to reference `store.$plugin`. 

Unfortunately the initialization code has circular dependencies between creating the router and created the store that will need to be untangled before this can be tackled.

Reply:
Could we also have an issue for this, unless you planning to work straight away?

Reply:
Todo added and issue created: https://github.com/rancher/dashboard/issues/11275